frontend/src/components/LogIn/login.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './login.module.css';
+import gmail_icon from '../Logos/gmail_icon.png';
+import fb_icon from '../Logos/fb_icon.png';
+import twtx_icon from '../Logos/twtx_icon.webp';
+import apple_icon from '../Logos/apple_icon.png';
+
+function EmailErrorMessage({username, isValid}) {
```

Comment:
For functions in javascript (not including naming the functional component) we use camel case.

Rename function to emailErrorMessage

Reply:
Technically this is a JSX React Component, so PascalCase is correct. But see my comment on this stuff, they should technically be in separate files, but i think we can hold off on that since this form will probably need to change when we do proper auth